.concourse/tasks/linux-book.yml

```diff
@@ -0,0 +1,19 @@
+---
+platform: linux
+
+image_resource:
+  type: docker-image
+  source:
+    repository: magnonellie/amethyst-dependencies
+    tag: stable
+
+inputs:
+- name: amethyst
+- name: target
+
+run:
+  path: sh
+  dir: amethyst
+  args:
+  - -exc
```

Comment:
tab here? (*stupid yaml*)

Reply:
Yaml formatting usually do it this way (without tabs for lists).

Reply:
Yaml is disgusting 😄

Reply:
I can change that for all yaml here if you want, but it seems the most commonly accepted way of writing yaml is the way I have done it currently.

Reply:
I believe tabs are actually not allowed in YAML? Anyway, the yaml formatting looks fine to me. If we want to have a consistent standard, we could just agree on the rules and use an auto formatter as part of the toolchain?

Reply:
I say leave it the way it is, it's the most common way of doing it.